acousticbrainz/views/api.py

```diff
@@ -7,11 +7,32 @@
 api_bp = Blueprint('api', __name__)
 
 
+@api_bp.route("/<uuid:mbid>/meta", methods=["GET"])
```

Comment:
Not sure about the /meta enpoint. Do you foresee returning other metadata here? Because if this is only to be used for a count then /count might be best.